samples/Integration/CMake-fetch-content-app/README.md

```diff
@@ -0,0 +1,42 @@
+# Integrating the Azure SDK for C++ with CMake fetch content to your application
+
+This application shows how to integrate the Azure SDK for C++ in your application. It uses CMake fetch content functionality to clone the Azure SDK for C++ repo into your `build` directory. This approach is useful when using CMake to build your application.
+
+## Pre-requisites
+
+Install the [Azure SDK for C++ dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#third-party-dependencies).
+
+- CMake project (min version 3.13).
+- C++ version 14 or greater.
+
+## Build
+
+```bash
+#
+# Building the application.
+# Instructions from application root directory.
+#
+
+# Create build directory just the first time.
+mkdir build
+cd build
+
+# Generate and build
+# This code assumes that the SDK dependencies were installed with VCPKG
+cmake -DCMAKE_TOOLCHAIN_FILE=path/to/vcpkg/scripts/buildsystems/vcpkg.cmake ..
```

Comment:
Is this the recommended (and only) way to let cmake know to pickup vcpkg or is there a way to set it up, so I don't have to repeatedly pass in `CMAKE_TOOLCHAIN_FILE`?

Reply:
it's a personal/developer preference.

Some projects do what the Azure SDK does and set the toolchain inside the CMake list. The bad thing about this is if someone wants to set a different value, they can't use command line, they need to update the CMake file.
The benefit is to avoid writing this command, however, you tipycally only need to write it once, then you would use the commands history to repeat.

If we see what VS does, it generates and set the toolchain as part of the generate script every time.

So, I don't think there's a right or wrong way to do this.  

For this case, I just want to make sure that the sample can be copy pasted and it would build/run